### PR TITLE
Issue #518

### PR DESCRIPTION
**Pull request by geetanshu-m**
Pull Request for Issue #518.

### Summary of Changes

Now The Title is not overflowing and, attributes of the files are not colliding with their values. 

### Testing Instructions

Checking the info for any file.

### CODE REVIEW

**Comment by kasvith** [comment]
Can you provide us some screenshots ?

**Comment by geetanshu-m** [comment]
screenshot 78

**Comment by laoneo** [comment]
Can you make the screenshot without the patch? That we see the difference? You need also to compile the sass changes trough npm run build in the folder /administrator/components/com_media. I'm sure @kasvith can help you on that :smile:

**Comment by ciar4n** [comment]
Thank you @proman24 

To compile the SASS you can run `npm run build` in the `/administrator/components/com_media` folder

If you haven't already done so run `npm i` first to install the needed packages.

I guess the question here is to decide do we shorten the file name with an ellipsis as @proman24 has done here or should we wrap the file name to a second line. Something like... 

image

**Comment by kasvith** [comment]
I like wrapping the name to next lines

**Comment by dneukirchen** [comment]
+1 for wrap

**Comment by geetanshu-m** [comment]
@ciar4n It's a great idea to wrap into a new line.

File: administrator/components/com_media/resources/styles/components/_media-infobar.scss [hunk 1]
```diff
@@ -11,6 +11,10 @@
   background-color: $info-bg;
   border-left: 1px solid $border-color;
   h2 {
```
Comment (ciar4n) [inline administrator/components/com_media/resources/styles/components/_media-infobar.scss:13]: @proman24 Would you like to make the changes here in this PR?.. 

```
word-wrap: break-word;
padding: 10px 40px 10px 15px;
```

Comment (geetanshu-m) [reply administrator/components/com_media/resources/styles/components/_media-infobar.scss:13]: @ciar4n Sure, would be updating this PR soon.

**Comment by geetanshu-m** [comment]
screenshot 84

**Comment by kasvith** [comment]
@proman24 why some files are deleted ?

Make sure you also do `git pull upstream master` before push. Then you can resolve merge conflicts.
To add this repo as upstream run 
`git remote add upstream https://github.com/joomla-projects/media-manager-improvement.git`

Also make sure you commit media/com_media/* files along with this. You need to build before commit using `npm run build`

**Comment by kasvith** [comment]
@proman24 i see an image with very_very_....long filename.
You may wanna delete that from commit (just remove from git)

File: administrator/components/com_media/resources/styles/components/_media-infobar.scss [hunk 2]
```diff
@@ -7,14 +7,15 @@
...
-  overflow: hidden;
+  overflow-y: scroll;
   background-color: $info-bg;
   border-left: 1px solid $border-color;
   h2 {
-    padding: 0 $gutter-width;
+    padding: 10px 40px 10px 15px;
     margin: (-$gutter-width) (-$gutter-width) $gutter-width;
     font-weight: normal;
     line-height: $info-title-height;
```
Comment (ciar4n) [inline administrator/components/com_media/resources/styles/components/_media-infobar.scss]: The line height is rather high. Could you reduce the value for this variable to say 2rem in the variables.scss... https://github.com/joomla-projects/media-manager-improvement/blob/dev/administrator/components/com_media/resources/styles/variables.scss#L65

File: administrator/components/com_media/resources/styles/components/_media-infobar.scss [hunk 3]
```diff
@@ -7,14 +7,15 @@
   float: none;
   width: 25%;
   padding: $gutter-width;
-  overflow: hidden;
+  overflow-y: scroll;
   background-color: $info-bg;
   border-left: 1px solid $border-color;
   h2 {
-    padding: 0 $gutter-width;
+    padding: 10px 40px 10px 15px;
```
Comment (ciar4n) [inline administrator/components/com_media/resources/styles/components/_media-infobar.scss]: Change to ``10px ($gutter-width + 30px) 10px $gutter-width;``

File: .github/CONTRIBUTING.md [hunk 4]
```diff
@@ -1,24 +0,0 @@
-Contributing to the Joomla! CMS™
```
Comment (dneukirchen) [inline .github/CONTRIBUTING.md]: Please revert the deletion

File: .github/ISSUE_TEMPLATE.md [hunk 5]
```diff
@@ -1,18 +0,0 @@
-### Steps to reproduce the issue
```
Comment (dneukirchen) [inline .github/ISSUE_TEMPLATE.md]: Please revert the deletion

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 6]
```diff
@@ -1,20 +0,0 @@
-Pull Request for Issue # .
```
Comment (dneukirchen) [inline .github/PULL_REQUEST_TEMPLATE.md]: Please revert the deletion

File: .github/SUPPORT.md [hunk 7]
```diff
@@ -1,8 +0,0 @@
-Where can you get support and help?
```
Comment (dneukirchen) [inline .github/SUPPORT.md]: Please revert the deletion

**Review by dneukirchen — CHANGES_REQUESTED** [review]

**Comment by laoneo** [comment]
Revert did not fully work with the deleted files. They are different. They must be exactly the same.

**Comment by geetanshu-m** [comment]
All files have been restored

**Comment by ciar4n** [comment]
Thank you @proman24

**Comment by laoneo** [comment]
Thanks for the help!

**Comment by kasvith** [comment]
Nice work @proman24

**Pull request merged**